Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -1,19 +1,20 @@
 import fs = require('fs');
+import os = require('os');
 import path = require('path');
 import taskLib = require('vsts-task-lib/task');
 import tr = require('vsts-task-lib/toolrunner');
-import toolLib = require('vsts-task-tool-lib/tool');
+import * as toolLib from 'vsts-task-tool-lib/tool';
 
 export class JavaFilesExtractor {
     public destinationFolder: string;
-    public win: RegExpMatchArray;
+    public win: Readonly<boolean>;
```

Comment:
You mean `public readonly win: boolean`.  `Readonly<boolean>` is actually a mapped type: https://www.typescriptlang.org/docs/handbook/advanced-types.html